### PR TITLE
move fit config modification to on-build-config lib

### PR DESCRIPTION
**Pull request by changev**

### CODE REVIEW

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
test this please

**Comment by panpan0000** [comment]
@PengTian0  to review

File: packer/ansible/roles/setuptools/tasks/main.yml [hunk 1]
```diff
@@ -4,3 +4,6 @@
 
 - pip: name=setuptools extra_args='--upgrade'
   sudo: yes
+
+- pip: name=virtualenv extra_args='--upgrade'
+  sudo: yes
```
Comment (PengTian0) [inline packer/ansible/roles/setuptools/tasks/main.yml]: Why upgrade virtualenv in this PR?

**Comment by PengTian0** [comment]
Please update the ```Usage``` according to your change

File: packer/ansible/roles/setuptools/tasks/main.yml (hunk 1 above)
Comment (changev) [reply packer/ansible/roles/setuptools/tasks/main.yml]: in ova venv version is too low.
And with such venv,  pip install  in ```vnc_record.sh``` will throw error.

Comment (PengTian0) [reply packer/ansible/roles/setuptools/tasks/main.yml]: I think it's better to modify it in PR for build_ova.sh

**Comment by changev** [comment]
test this please

File: packer/ansible/roles/setuptools/tasks/main.yml (hunk 1 above)
Comment (changev) [reply packer/ansible/roles/setuptools/tasks/main.yml]: but without the ova with upgraded virtualenv, current ova post test won't work.

one pr or two pr is both ok for me ~  but I think this is not a urgent topic, merge this PR and let feature branch work is the urgent affairs

**Comment by panpan0000** [comment]
👍 

@iceiilin  to review

**Pull request merged**